### ISSUE TITLE
Support SVG attribute - r

### ISSUE THREAD

**Issue by kagawagao** [stylelint/stylelint#2119]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

New feature: support `SVG` attribute - `r`

> Which rule, if any, is this issue related to?

`property-no-unknown`

> What CSS is needed to reproduce this issue?

```css
.circle {
  r: 10px;
  fill: #fff;
  stroke: #48adff;
  stroke-width: 2px;
  transition: all 0.3s ease;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "extends": "stylelint-config-standard"
}
```

> Which version of stylelint are you using?

 `7.5.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint src/themes/**/*.css`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to `PostCSS`.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
src/themes/default/views/chart/chart.css
 22:13  ×  Unexpected unknown property "r"   property-no-unknown
 29:15  ×  Unexpected unknown property "r"   property-no-unknown
 43:13  ×  Unexpected unknown property "r"   property-no-unknown
 50:15  ×  Unexpected unknown property "r"   property-no-unknown
 64:13  ×  Unexpected unknown property "r"   property-no-unknown
 71:15  ×  Unexpected unknown property "r"   property-no-unknown
 85:13  ×  Unexpected unknown property "r"   property-no-unknown
 92:15  ×  Unexpected unknown property "r"   property-no-unknown
```

**Comment by evilebottnawi** [comment]
@kagawagao `property-no-unknown` use https://github.com/betit/known-css-properties, please not create issue here, create this in https://github.com/betit/known-css-properties repo. Thanks for helping us!

**Comment by evilebottnawi** [comment]
Issue: https://github.com/betit/known-css-properties/issues/6

**Comment by kagawagao** [comment]
sorry, I didn't notice that, thanks !!!

**Comment by jeddy3** [comment]
@kagawagao In the meantime you can use the rule's [`ignoreProperties`](https://github.com/stylelint/stylelint/blob/master/src/rules/property-no-unknown/README.md#ignoreproperties-regex-string) secondary option to work around the issue.

**Comment by kagawagao** [comment]
@jeddy3 👍

**Comment by jeddy3** [comment]
FYI, a new version of `known-css-properties` (containing `r` etc...) will be [bundled in](https://github.com/stylelint/stylelint/pull/2132) the next stylelint release.
